spec/features/idv/doc_auth/redo_document_capture_spec.rb

```diff
@@ -108,10 +108,9 @@
 
       it 'shows a troubleshooting option to allow the user to cancel and return to SP' do
         click_idv_continue
-
         expect(page).to have_link(
           t('links.cancel'),
-          href: idv_cancel_path,
+          href: idv_cancel_path(step: :session, location: :warning),
```

Comment:
Just curious: is there a reason we had to add a `location` here and to `unavailable` and not to the others?

Reply:
@night-jellyfish, not really, from what i see the location mostly means where the link is on the page, some easy to tell, some not.

Reply:
Interesting! I imagine this would be useful when trying to determine where someone cancelled from.

If so, could we add it to some of the others we're changing here, like `app/views/sign_up/completions/show.html.erb`, since we know that location?

If this is out of scope, I understand. I just thought that if we're adding it in some places, it might be helpful to add it to others as well.